### PR TITLE
Update the environment creation to just use conda and use python 3.11

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
In this PR I update the environment creation part of the installation instructions to:
1. just use conda, rather than mix and match with pip
2. eliminate special M1 instructions, which aren't needed anymore
3. use jupyterlab to match the rest of the materials
4. use napari-workshop as the name of the env for clarity and consistency

### CODE REVIEW

File: napari-workshops/installation.md [hunk 1]
```diff
@@ -98,53 +98,43 @@ as well and you can skip to the next section.
 2. We use an environment to encapsulate the Python tools used for this workshop.
    This ensures that the requirements for this workshop do not interfere with
    your other Python projects. To create the environment (named
-   `napari-tutorial`) and install Python 3.9 in it, enter the following command:
+   `napari-workshop`) and install Python 3.11 and napari in it, enter the following command:
```
Comment (psobolewskiPhD) [inline napari-workshops/installation.md:101]: This is to match the env.yml and using 3.11 is just better now with better error messages and bit faster.

File: napari-workshops/installation.md [hunk 2]
```diff
@@ -98,53 +98,43 @@ as well and you can skip to the next section.
...
 
     ```bash
-    conda create -n napari-tutorial python=3.9
+    conda create -n napari-workshop -c conda-forge python=3.11 napari pyqt jupyterlab
```
Comment (psobolewskiPhD) [inline napari-workshops/installation.md:104]: I think it's easier to just install napari and jupyter directly, every workshop will need that I presume.

File: napari-workshops/installation.md [hunk 3]
```diff
@@ -98,53 +98,43 @@ as well and you can skip to the next section.
...
+4. Install the additional workshop dependencies with the commands below.  
 
-    **If you're on an M1 Mac**:
+   For the plugin template:
 
     ```bash
-    conda install -c conda-forge notebook napari
-    python -m pip install cookiecutter magicgui
-    python -m pip install stardist-napari
+    conda install -c conda-forge cookiecutter
```
Comment (psobolewskiPhD) [inline napari-workshops/installation.md:121]: napari pulls in magicgui, so we don't need that as it's own thing and it's best to just stick to conda for everything IMO.

File: napari-workshops/installation.md [hunk 4]
```diff
@@ -98,53 +98,43 @@ as well and you can skip to the next section.
...
-    conda install -c conda-forge notebook
-    python -m pip install cookiecutter magicgui "napari[all]"
-    python -m pip install stardist-napari
-    ```
-
-5. If you are on a Mac, please install this one additional dependency.
-
-    ```python
-    conda install -c conda-forge python.app
+    conda install -c conda-forge stardist-napari
```
Comment (psobolewskiPhD) [inline napari-workshops/installation.md:127]: stardist is a bit heavy, so making this it's own install is best I think? Again, using just conda-forge nothing special is needed.

File: napari-workshops/installation.md [hunk 5]
```diff
@@ -156,33 +146,9 @@ as well and you can skip to the next section.
     ```
 
 ````{admonition} Errors launching?
-Sometimes, `napari` installation can fail on an M1 Mac due to mismatching
-dependencies on `pip`.
```
Comment (psobolewskiPhD) [inline napari-workshops/installation.md:160]: We don't mix conda and pip anymore, so I think this stuff can be deleted -- I run exclusively on M1 macOS without issues.

**Review by jni — APPROVED** [review]
Great! Thanks @psobolewskiPhD!

I'm pro-micromamba in general but I understand that being less common it has its downsides. I'm gonna merge! 🚀

**Pull request merged**